### PR TITLE
Build action for GH pages

### PR DESCRIPTION
**Pull request by glebovsky**

### CODE REVIEW

File: .github/workflows/webpack.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: Build Metronome
+
+on:
+  push:
+    branches: [ master ]
```
Comment (marcua) [inline .github/workflows/webpack.yml:5]: i'm excited for us to change to `main` in a subsequent step

File: .github/workflows/webpack.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        node-version: '14'
+      
+    - name: Install and Build 🔧
+      run: |
+        yarn install
+        yarn build
+
+    - name: Deploy 🚀
+      uses: JamesIves/github-pages-deploy-action@v4.2.5
+      with:
```
Comment (marcua) [inline .github/workflows/webpack.yml:29]: i see! let's see if this does the trick. if we need to, we can always switch to `gh-branches` if this doesn't work

File: .github/workflows/webpack.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+name: Build Metronome
```
Comment (marcua) [inline .github/workflows/webpack.yml:1]: proposed rename: `build-website.yml`?

File: .github/workflows/webpack.yml (hunk 2 above)
Comment (glebovsky) [reply .github/workflows/webpack.yml:29]: Okie!

**Review by marcua — APPROVED** [review]

**Pull request merged**